pennylane/operation.py

```diff
@@ -1025,6 +1029,29 @@ def generator(self):  # pylint: disable=no-self-use
         """
         raise GeneratorUndefinedError(f"Operation {self.name} does not have a generator")
 
+    def pow(self, n):
+        """A new operator equal to this one raised to the given power.
+
+        This *developer* method should be used to define shortcuts and easier decompositions.
+        For example, this method may specify how a Pauli gate raised to an even power is equal
+        to nothing, or we can exponentiate a rotation gate by multiplying the angle by the exponent.
+        This method should *not* return default behavior like a list of the same gate many times.
```

Comment:
I'm not sure this is clear. What do we want to avoid here exactly? That the identity is represented as `PauliX*PauliZ`?